spec.md

```diff
@@ -600,6 +612,147 @@ message Volume {
   // the same attributes. This field is OPTIONAL and when present MUST
   // be passed to volume validation and publishing calls.
   map<string,string> attributes = 3;
+
+  // The topological segment(s) the provisioned volume is accessible
+  // from. A topological segment is a sub-division of a cluster, like
+  // region, zone, rack, etc.
+  // Each key must consist of alphanumeric characters, '-', '_' or '.'.
+  // Each value MUST contain 1 or more valid strings.
+  // A value containing more than 1 string indicates the volume is
+  // accessible from multiple topological segments.
+  // COs may use this information along with the topology information
+  // returned by NodeGetId to ensure that a given volume is accessible
+  // from a given node when scheduling workloads.
+  // This field is OPTIONAL. If it is not specified, the CO MAY assume
```

Comment:
How are node-local volumes accounted for by this strategy? If this field is OPTIONAL but the volume is local (is tied to a particular node) then it doesn't make sense that it's "equally accessible from all nodes in the cluster".

Reply:
Node local volumes should set a topological segment {"node": {"nodeName"}} and this would have to match between node and volume.